lib/Imgur/Auth/OAuth2.php

```diff
@@ -12,8 +12,8 @@
  */
 class OAuth2 implements AuthInterface
 {
-    const AUTHORIZATION_ENDPOINT = 'https://api.imgur.com/oauth2/authorize';
-    const ACCESS_TOKEN_ENDPOINT = 'https://api.imgur.com/oauth2/token';
+    public const AUTHORIZATION_ENDPOINT = 'https://api.imgur.com/oauth2/authorize';
+    public const ACCESS_TOKEN_ENDPOINT = 'https://api.imgur.com/oauth2/token';
```

Comment:
This syntax is only available on `php-7.1` at least.

Reply:
Please look at the [Example #4](https://www.php.net/manual/en/language.oop5.constants.php).

Reply:
Yep already reverted